test/ad/distributions.jl

```diff
@@ -395,6 +395,11 @@
             # Broken distributions
             d.f(d.θ...) isa Union{VonMises,TriangularDist} && continue
 
+            # Skellam only fails in these tests with ReverseDiff
```

Comment:
Please open an issue with this.

Reply:
I opened an issue: https://github.com/TuringLang/DistributionsAD.jl/issues/126